.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
 <!-- Thanks so much for your PR, your contribution is appreciated! ❤️ -->
 
-- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/material-ui/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).
+- Fixes <!-- Issue hyperlink -->
+
+- Visual demonstration <!-- Screenshot/Video link -->
+
+- Please look at the [PR section of the contributing guide](https://github.com/mui/material-ui/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request) and also MUI's broad [guidelines](https://www.notion.so/mui-org/GitHub-PRs-7112d03a6c4346168090b29a970c0154#258c9e963dc445348562a52a62052941) on Pull Requests.
```

Comment:
I'm not a fan of having multiple links here because it's something that we will need to keep up to date. At the same time, I believe that we need the same information in https://github.com/mui/mui-toolpad/blob/master/CONTRIBUTING.md.

So to solve this problem, I would propose to link the relevant section in https://github.com/mui/mui-toolpad/blob/master/CONTRIBUTING.md, that could in turn, defer to the notion and material ui docs.